### DISCUSSION TITLE
Why is Steam available in the Versions bucket but not in Extras?

### DISCUSSION THREAD

**Discussion by mariobrostech** [ScoopInstaller/Scoop#5399] [Help and Questions] [answered]
Hi,

I was looking into setting up Scoop earlier today and migrating to it, but I made an interesting discovery during this process: Steam is only present in the Versions bucket. I found this a bit strange, so I did some more searching and wasn't able to find any similar apps that I could think of where this was the case. So, I guess my question is what constitutes such a decision? It's my understanding that open-source, CLI tools are present in the Main bucket, while proprietary, GUI-centric apps are in the Extras bucket, with the Versions bucket just being used for additional versions of already published applications (such as Chrome Canary or Firefox Nightly). Is this correct, or am I missing something?

Thanks in advance!

**Comment by niheaven** [comment, marked as answer]
Beta (non stable) version (incl. nightly version) is also in versions bucket, too. Maybe Steam is such a case?

**Reply by mariobrostech** [reply]
Possibly, but the Steam client also has regularly released stable updates as well. Is there a reason for Scoop not including these releases as a package in the Extras bucket?

**Reply by niheaven** [reply]
Right? You could submit a package request issue under `extras`, and I or someone else will create it.

**Reply by mariobrostech** [reply]
Ok, thanks! I didn't initially want to open an issue because I wasn't sure if there was a specific reason for it being available in only one bucket and not the other. I'll open a package request shortly, in that case 🙂
